lib/ExpenseRule.jsx

```diff
@@ -21,11 +21,7 @@ export default class ExpenseRule {
      * @return {Object}
      */
     getApplyWhenByField(field) {
-        return (
-            _.find(this.applyWhen, (conditions) => {
-                return conditions.field === field;
-            }) || {}
-        );
+        return _.find(this.applyWhen, (conditions) => conditions.field === field) || {};
```

Comment:
Why underscore?

Reply:
Because it's a JS file, and underscore and lodash is removed only from TS files

Reply:
It's a JS file, and we only remove `underscore`/`lodash` from TS files, right?

Reply:
oh, sorry I missed that 🤦🏼 you're right